eXolnet/redmine-pulls
# Enhance branch validation when creating a pull request

# Description


Fixes #7 

## Type of change

# How Has This Been Tested?
## Test A – Same branches
1. Create a new pull request
2. Choose `master` as base branch
3. Choose `master` as compare branch
4. Enter en random subject
5. Hit `Create`
6. You should see this error: `You can't create a pull request based on the same branch.`

## Test B – Non existing branch
1. Create a new pull request
2. Enter `foobar` as base branch
3. Enter `example` as compare branch
4. Enter en random subject
5. Hit `Create`
6. You should see these errors:
6.1 `Branch "foobar" doesn't exist.`
6.2 `Branch "example" doesn't exist.`

# Checklist:
- [ ] `Test A` has been done
- [ ] `Test B` has been done
